LayoutTests/http/wpt/fetch/fetch-as-blob.js

```diff
@@ -66,7 +66,7 @@ promise_test(function(t) {
     req.headers.set('Content-Type', 'Text/Html');
     return req.blob()
       .then(function(blob) {
-          assert_equals(blob.type, 'text/plain');
+          assert_equals(blob.type, 'text/html');
```

Comment:
I just updated the test here to confirm there's no other issues. But, I've sent a PR to add this test to upstream wpt so I will import that upstream tests and remove this one when the upstream PR is merged.

Reply:
I've reviewed and merged https://github.com/web-platform-tests/wpt/pull/39489.

The WebKit changes look reasonable to me, but @youennf will need to do a final review.

Reply:
Thank you so much for your help to merge the WPT PR 🙏 
I will import them and update this PR later 🙇